apps/minifront/src/state/send/index.ts

```diff
@@ -84,18 +88,23 @@ export const createSendSlice = (): SliceCreator<SendSlice> => (set, get) => {
       if (!amount || !recipient || !selection) {
         set(state => {
           state.send.fee = undefined;
+          state.send.assetFeeMetadata = undefined;
         });
         return;
       }
 
       const txPlan = await plan(assembleRequest(get().send));
       const fee = txPlan.transactionParameters?.fee;
-      if (!fee?.amount) {
-        return;
-      }
+
+      const feeAssetMetadata = fee?.assetId
+        ? await getAssetTokenMetadata(fee.assetId)
+        : await getStakingTokenMetadata();
+
+      if (!fee?.amount) return;
```

Comment:
`fee?.assetId` will return `undefined` when the user has UM, since we opt-out of storing the native staking token assetId in the naive case. In that context, we need to query the staking token metadata from the registry.